packages/data-flow/src/orchestrator.ts

```diff
@@ -113,16 +113,25 @@ export class Orchestrator {
                 await this.eventsRegistry.saveLastProcessedEvent(this.chainId, event);
 
                 event = await this.enhanceStrategyId(event);
-                if (event.contractName === "Strategy" && "strategyId" in event) {
+                if (this.isPoolCreated(event)) {
+                    // we save the strategy id with handled = false because we don't know if the strategy is handled yet
+                    await this.strategyRegistry.saveStrategyId(
+                        this.chainId,
+                        event.srcAddress,
+                        event.strategyId,
+                        false,
+                    );
+                } else if (event.contractName === "Strategy" && "strategyId" in event) {
                     if (!existsHandler(event.strategyId)) {
-                        //TODO: save to registry as unsupported strategy, so when the strategy is handled it will be backwards compatible and process all of the events
-                        //TODO: decide if we want to log this
-                        // this.logger.info(
-                        //     `No handler found for strategyId: ${event.strategyId}. Event: ${stringify(
-                        //         event,
-                        //     )}`,
-                        // );
+                        // we skip the event if the strategy id is not handled yet
```

Comment:
shouldn't we save it with `handled=false` here ?

Reply:
maybe `handleable` is more representative than `handled`

Reply:
we don't need to, we register on every PoolCreated the strategyAddress to strategyId entry with `handled=false` since it's only the pool creation. then when an event from the Strategy arrives, if it can be handled then we update the `handled=true`, we don't need to mark it again as handled=false